code-of-conduct.md

```diff
@@ -0,0 +1,61 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of
+fostering an open and welcoming community, we pledge to respect all people who
+contribute through reporting issues, posting feature requests, updating
+documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
+experience for everyone, regardless of level of experience, gender, gender
+identity and expression, sexual orientation, disability, personal appearance,
+body size, race, ethnicity, age, religion, or nationality.
```

Comment:
Adding some from Microsoft's CoC.  culture, color, social and economic class, educational level, family status, political belief

Reply:
```suggestion
body size, race, ethnicity, age, religion, nationality, socio-economic status, educational level, 
family status, culture, or political belief.
```

I believe "color" is already covered in "race", "ethnicity" and "personal appearance".

We should also note that this list is not exhaustive and it would be impossible to be exhaustive.